lib/utils/version.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright 2018-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as semver from 'semver';
+import { version } from '../../package.json';
+
+export function isVersionGTE(v: string): boolean {
```

Comment:
I came across a module import loop when `messages.ts` imported `isVersionGTE` from `helpers.ts`, which imported `errors.ts`, which imported `messages.ts`. Hence moving it to its own `version.ts` module.